src/components/LoginForm.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Field, reduxForm } from 'redux-form'
+import { Redirect, Link } from 'react-router-dom'
+import { Button, Icon } from 'semantic-ui-react'
+import { renderСommonTextInput } from './inputComponents/inputComponents'
+import '../styles/login.css'
+
+const LoginForm = props => {
+  const { handleSubmit, submitting } = props
+  return (
+    <form className="login__form" onSubmit={handleSubmit}>
```

Comment:
onSubmit={ handleSubmit }. You should write your code in one style. You declare imports on the top of the file using { … } syntax.